components/footer/footer.tsx

```diff
@@ -21,9 +21,9 @@ const classes = {
 export const Footer = () => {
   return (
     <footer className={`${TopBar()} mt-28`}>
-      <div className="container mx-auto" style={{ padding: '10px 0' }}>
+      <div className="container mx-auto" style={{ padding: '20px 0' }}>
         <div className="flex flex-col justify-center md:flex-row md:justify-between">
-          <span className="text-center">Barranquilla JS (BAQJS) - {new Date().getFullYear()}</span>
+          <h3 className="text-center">Barranquilla JS (BAQJS) - {new Date().getFullYear()}</h3>
```

Comment:
Sticky header floating in all the document could have negative impact with the semantic order of titles `hn`

Reply:
Sounds ok for me